scholarx/2023/functions.js

```diff
@@ -0,0 +1,69 @@
+// Write custom js functions for the scholarx 2020 page here
+
+
+$(function () {
```

Comment:
can't we use the same function file in 2022? @Madhawa97

Reply:
We can. But I was assuming we may have to move the /2022 to /past-programs. I thought it's better to have a separate one for this.

Reply:
we can have a separate one when we have specific scripts for 2023 one only.

Reply:
Alrighty! Pushed the changes @anjula-sack